pkg/disttask/framework/integrationtests/framework_ha_test.go

```diff
@@ -82,24 +83,24 @@ func TestHARandomShutdownInDifferentStep(t *testing.T) {
 	submitTaskAndCheckSuccessForHA(c.Ctx, t, "😊", c.TestContext)
 }
 
-func TestHAReplacedButRunning(t *testing.T) {
```

Comment:
those cases depends on `mockTiDBPartitionThenResume` which only remove the node from "PD" and pause execution temporaryly, no network is partitioned, and there's no multiple owner, so remove them